src/Sylius/Bundle/AdminBundle/Resources/config/app/template_events.yaml

```diff
@@ -0,0 +1,54 @@
+# This file is part of the Sylius package.
+# (c) Sylius Sp. z o.o.
+
+imports:
+    - { resource: 'template_events/*.yaml' }
+
+sylius_ui:
```

Comment:
Different names for `syiius_ui` bundle and `template_events` files will be a source of confusion.

Reply:
we'll unify that at some point, we'll go with `ui events` or `template events` as we have `sylius_template_event` and `sylius_ui.events` 🤷🏼‍♂️.